about.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>About Me</title>
+</head>
+<body>
+    <nav>
+        <h1><a href="/index.html">Adelle's Smoothies</a></h1>
+        <ul>
+            <li><a href="/index.html">Home</a></li>
```

Comment:
I noticed that the links in your site weren't quite working (they might on your computer, but they dont on mine). If you add a . in front of your slash, you can use relative file paths and it should work better! change your links from <a href="/index.html"> to <a href="./index.html">

More info about relative file paths
https://www.w3schools.com/html/html_filepaths.asp
https://stackoverflow.com/questions/24028561/relative-path-in-html